weaver/core/relay/Cargo.toml

```diff
@@ -11,16 +11,23 @@ path = "src/main.rs"
 [[bin]]
 name = "client"
 path = "src/client.rs"
+
+[[bin]]
+name = "satp_client"
+path = "src/satp_client.rs"
+
 [[bin]]
 name = "client-tls"
 path = "src/client_tls.rs"
 
+[[bin]]
```

Comment:
this is duplicate entry as line 15-17 also points to same file `src/satp_client.rs`

Reply:
Good spot. Fixed.